web/docs/cli.md

```diff
@@ -80,7 +81,7 @@ To setup Bash completion, execute `wasp completion` and follow the instructions.
 
 ### In project
  - `wasp start` runs Wasp app in development mode. It opens a browser tab with your application running, and watches for any changes to .wasp or files in `src/` to automatically reflect in the browser. It also shows messages from the web app, the server and the database on stdout/stderr.
-
+ - `wasp start db` start the database for you -> this can be very handy, since you don't need to spin up your own database or provide its connection URL to the Wasp app!
```

Comment:
```suggestion
 - `wasp start db` starts the database for you. This can be very handy, since you don't need to spin up your own database or provide its connection URL to the Wasp app!
```

Reply:
Done!